lib/msf/base/serializer/readable_text.rb

```diff
@@ -555,7 +556,11 @@ def self.dump_sessions(framework, opts={})
       if session.respond_to? :platform
         row[1] += " " + session.platform
       end
-      row << session.via_exploit if verbose and session.via_exploit
+
+      if verbose
+        row << session.via_exploit.to_s
+        row << session.payload_uuid.to_s
```

Comment:
Will `via_exploit` or `payload_uuid` ever be `nil`? Should we do this instead, to be safe:
``` ruby
row << (session.via_exploit || '').to_s
row << (session.payload_uuid || '').to_s
```
I'm guessing `payload_uuid` will be fine, given the changes we're making here. But the history of `via_exploit` had a check for `nil` (even if it was incorrect!). Should we be maintaining it as above?